src/PathTranslator.ts

```diff
@@ -113,9 +128,10 @@ export class PathTranslator {
 		const possiblePaths = new Array<string>();
 		const pathInfo = PathInfo.from(filePath);
 
-		// index.*.lua cannot come from a .ts file
-		if (pathInfo.extsPeek() === LUA_EXT && pathInfo.fileName !== INDEX_NAME) {
-			pathInfo.exts.pop();
+		// index.*.lua(u) cannot come from a .ts file
+		if (pathInfo.extsPeek() === this.getLuauExt() && pathInfo.fileName !== INDEX_NAME) {
+			const originalExt = pathInfo.exts.pop();
+			assert(originalExt);
```

Comment:
This variable seems unnecessary if we only ever check for one extension. Maybe a remnant from earlier changes?